src/plugins/interaction/GraphInteraction/CMakeLists.txt

```diff
@@ -12,6 +12,8 @@ add_library(${LIBRARY_NAME} SHARED
   ${SRCS}
 )
 
+include_directories(${Boost_INCLUDE_DIRS})
```

Comment:
Can we use `target_include_directories` instead?

Reply:
Sure, though please double-check my use of it in upcoming push; not quite sure whether my use of it covers both the build and install tree parts correctly. Thanks!